src/pages/New-Beer.jsx

```diff
@@ -1,14 +1,81 @@
-import React from 'react'
-import Header from '../components/Header'
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import axios from 'axios';
+import Header from '../components/Header';
 
 function NewBeer() {
-  return (
-    <div>New-Beer
+  const [name, setName] = useState("")
+  const [tagline, setTagline] = useState("")
+  const [description, setDescription] = useState("")
+  const [firstBrewed, setFirstBrewed] = useState("")
+  const [brewersTips, setBrewersTips] = useState("")
+  const [attenuationLevel, setAttenuationLevel] = useState(0)
+  const [contributedBy, setContributedBy] = useState("")
+
+  const navigate = useNavigate();
+
+  const handleName  = (e) => setName(e.target.value);
+  const handleTagline  = (e) => setTagline(e.target.value);
+  const handleDescription  = (e) => setDescription(e.target.value);
+  const handleFirstBrewed   = (e) => setFirstBrewed(e.target.value);
+  const handleBrewersTips  = (e) => setBrewersTips(e.target.value);
+  const handleAttenuationLevel  = (e) => setAttenuationLevel(e.target.value);
+  const handleContributedBy  = (e) => setContributedBy(e.target.value);
 
-    <Header />
-    
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const newBeer = {
+      name: name,
+      tagline:tagline,
+      description: description,
+      firstBrewed: firstBrewed,
+      brewersTips: brewersTips,
+      attenuationLevel: attenuationLevel,
+      contributedBy: contributedBy
```

Comment:
Your `new-beer` page is not properly working because you didn't use the `keywords` that the `api` is expecting:
```
    const newBeer = {
      name: name,
      tagline: tagline,
      description: description,
      first_brewed: firstBrewed,
      brewers_tips: brewersTips,
      attenuation_level: attenuationLevel,
      contributed_by: contributedBy
    }
```